src/components/current/MessageTranslator.tsx

```diff
@@ -186,6 +186,8 @@ export const Widget = (props: WidgetProps) => {
       tokenInSymbol={parsedArgs[0]}
       tokenOutSymbol={parsedArgs[1]}
       inputAmount={parsedArgs[3]}
+      buyOrSellAmount={parsedArgs[2]} // BUYAMOUNT or SELLAMOUNT
```

Comment:
might be a confusing var name (implies an amount/number).

we are using `transactionKeyword` or similar on the [backend](https://github.com/yieldprotocol/cacti-backend/pull/256/files#diff-78abba601e0374590ab195f4bfc61580d73854059ef5c63639831706f02650f1R579)